includes/core-changes/corefx/5.0/lastindexof-improved-handing-of-empty-values.md

```diff
@@ -0,0 +1,59 @@
+### LastIndexOf has improved handling of empty search strings
```

Comment:
nit: File name has a typo (handing vs handling). It's not very important to fix as it doesn't appear in the URL.